packages/eas-cli/src/commands/workflow/runs.ts

```diff
@@ -0,0 +1,135 @@
+import { AppQuery } from '../../graphql/queries/AppQuery';
+import Log from '../../log';
+import formatFields from '../../utils/formatFields';
+
+export type WorkflowRunResult = {
+  id: string;
+  status: string;
+  gitCommitMessage?: string | null;
+  gitCommitHash?: string | null;
+  startedAt: string | null;
+  finishedAt: string | null;
+  workflowId: string;
+  workflowName: string | null;
+  workflowFileName: string | null;
+};
+
+function processWorkflowRuns(
+  runs: AppWorkflowRunsFragment['runs'],
+  params: {
+    workflowId?: string | undefined;
+    workflowFileName?: string | undefined;
+    status?: string | undefined;
+  }
+): WorkflowRunResult[] {
+  const { workflowId, workflowFileName, status } = params;
+  return runs.edges
+    .filter(edge => {
+      if (workflowId && edge.node.workflow.id !== workflowId) {
+        return false;
+      }
+      if (workflowFileName && edge.node.workflow.fileName !== workflowFileName) {
+        return false;
+      }
+      if (status && edge.node.status !== status) {
+        return false;
+      }
+      return true;
+    })
+    .map(edge => {
+      const finishedAt =
+        edge.node.status === WorkflowRunStatus.InProgress ? null : edge.node.updatedAt;
+      return {
+        id: edge.node.id,
+        status: edge.node.status,
+        gitCommitMessage: edge.node.gitCommitMessage,
+        gitCommitHash: edge.node.gitCommitHash,
+        startedAt: edge.node.createdAt,
+        finishedAt,
+        workflowId: edge.node.workflow.id,
+        workflowName: edge.node.workflow.name ?? null,
+        workflowFileName: edge.node.workflow.fileName ?? null,
+      };
+    });
+}
+
+export default class ProjectWorkflowRunList extends EasCommand {
+  static override description =
+    'list recent workflow runs for this project, with their IDs, statuses, and timestamps';
+
+  static override flags = {
+    /*
+    workflowId: Flags.string({
+      description:
+        'If present, filter the returned runs to select those for the specified workflow ID',
+      required: false,
+    }),
+     */
+    workflow: Flags.string({
+      description:
+        'If present, filter the returned runs to select those for the specified workflow file name',
+      required: false,
+    }),
+    status: Flags.string({
+      description: 'If present, filter the returned runs to select those with the specified status',
+      required: false,
+      options: Object.values(WorkflowRunStatus),
+    }),
+    ...EasJsonOnlyFlag,
+    limit: getLimitFlagWithCustomValues({ defaultTo: 10, limit: 100 }),
+  };
+
+  static override contextDefinition = {
+    ...this.ContextOptions.ProjectId,
+    ...this.ContextOptions.LoggedIn,
+  };
+
+  async runAsync(): Promise<void> {
+    const { flags } = await this.parse(ProjectWorkflowRunList);
+    const {
+      projectId,
+      loggedIn: { graphqlClient },
+    } = await this.getContextAsync(ProjectWorkflowRunList, {
+      nonInteractive: true,
+    });
+
```

Comment:
add call similar to other commands:
```
if (flags.json) {
      enableJsonOutput();
    }
```
then use `printJsonOnlyOutput` for JSON output.

(ditto for list command)

Reply:
Same comment applies